tasks/build.py

```diff
@@ -19,11 +19,14 @@
 from collections import namedtuple
 from connections import github
 from datetime import datetime, timezone
+from distutils.dir_util import copy_tree
```

Comment:
OK to use for now, but `distutils` is deprecated since Python 3.10, and will be removed in Python 3.12, so we should open an issue to follow up and make sure we don't rely on `distutils` at all going forward.